docs/content/docs/recipes/react.md

```diff
@@ -5,247 +5,211 @@ aliases:
-
-![Four browsers with the Timestamp app open in them one second after the button has been pushed.](https://fluidframework.blob.core.windows.net/static/images/Four-clients-2.PNG)
+![cra](https://user-images.githubusercontent.com/1434956/111496992-faf2dc00-86fd-11eb-815d-5cc539d8f3c8.gif)
 
 {{< callout note >}}
 
 This tutorial assumes that you are familiar with the [Fluid Framework Overview]({{< relref "/docs/_index.md" >}}) and that you have completed the [QuickStart]({{< relref "quick-start.md" >}}). You should also be familiar with the basics of [React](https://reactjs.org/), [creating React projects](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app), and [React Hooks](https://reactjs.org/docs/hooks-intro.html).
 
 {{< /callout >}}
 
-## Create the project
+Concepts you will learn:
+1. How to integrate Fluid into a React application
+2. How to run and connect your application to a local Fluid service (Tinylicious)
+3. How to create and get Fluid Containers and collaborative objects
+4. How to use a [SharedMap distributed data structure (DDS)](https://fluidframework.com/docs/apis/map/sharedmap/) to sync data between connected clients
 
-1. Open a Command Prompt and navigate to the parent folder where you want to create the project; e.g., `c:\My Fluid Projects`.
-1. Run the following command at the prompt. (Note that the CLI is np**x**, not npm. It was installed when you installed Node.js.)
 
-    ```dotnetcli
-    npx create-react-app fluid-react-tutorial --use-npm
-    ```
+\* Just want to see the code? Jump to the [finished tutorial.](https://github.com/microsoft/FluidExamples/blob/main/cra-demo/src/App.js).
 
-1. The project is created in a subfolder named `fluid-react-tutorial`. Navigate to it with the command `cd fluid-react-tutorial`.
-1. The project uses two Fluid libraries:
+## Demo introduction
 
-    |Library |Description |
-    |---|---|
-    |fluid&#x2011;experimental/fluid&#x2011;framework&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;    |Contains the SharedMap [distributed data structure]({{< relref "dds.md" >}}) that synchronizes data across clients. *This object will hold the most recent timestamp update made by any client.*|
-    |fluidframework/azure&#x2011;client&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;   |Defines the connection to a Fluid service server and defines the starting schema for the [Fluid container][]. In this tutorial, we will use a local test service called Tinylicious.|
-    &nbsp;
+In this example you will do the following:
 
-    Run the following command to install the libraries.
+  - [Use Create React App](use-create-react-app)
+  - [Install Fluid package dependencies](install-fluid-package-dependencies)
+  - [Import and initialize Fluid dependencies](import-and-initialize-fluid-dependencies)
+  - [Get the Fluid SharedMap](get-the-fluid-sharedmap)
+  - [Update the view](update-the-view)
```

Comment:
```suggestion
  - [Use Create React App](#use-create-react-app)
  - [Install Fluid package dependencies](#install-fluid-package-dependencies)
  - [Import and initialize Fluid dependencies](#import-and-initialize-fluid-dependencies)
  - [Get the Fluid SharedMap](#get-the-fluid-sharedmap)
  - [Update the view](#update-the-view)
```

Reply:
weird.....i wonder why those dissapeared

Reply:
yeah, broken in example repo. fixed there.